components/logins/ffi/Cargo.toml

```diff
@@ -12,6 +12,7 @@ crate-type = ["lib", "staticlib", "cdylib"]
 serde_json = "1.0.28"
 log = "0.4"
 url = "1.7.1"
+base16 = "0.1.1"
```

Comment:
Is it really worth a new dep to base16 encode?

Reply:
It's already what we use in the sync15 crate, so it's not actually a new dep.